tsconfig.json

```diff
@@ -4,9 +4,9 @@
     "module": "commonjs",
     "noEmit": true,
     "strict": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
I think this is true by default with `strict: true`? try removing it and see if stuff still works.